sprockets/mixins/http/__init__.py

```diff
@@ -495,3 +497,25 @@ def _http_req_user_agent(self):
             except AttributeError:
                 pass
         return DEFAULT_USER_AGENT
+
+    def _http_req_modify_for_retry(self, response: HTTPResponse, attempt: int,
```

Comment:
Why leading underscore? It tells me that this isn't part of this library's stable API.

Reply:
The rest of the subclass API is prefixed with `_http_req`. That seems to be the convention here.